### ISSUE TITLE
Counting Total Number of Objects detected using tf_od_predict.py?

### ISSUE THREAD

**Issue by Geo-Trackers** [developmentseed/label-maker#131]
Dear All, 

Has anyone worked on counting the total number of detected objects using tf_od_record.py ?  

Thanks in advance.

Best, 
Suman

**Comment by Geoyi** [comment]
yes, @Geo-Trackers, go to these few lines from [here](https://github.com/developmentseed/label-maker/blob/a057f5d2759d7f21646f55a51bd9f4c332d399e3/examples/utils/tf_od_predict.py#L61). 

You can write a simple helper function to loop through and write down each bbox, the associated class, and the tile ID to either a dataframe, list or dictionary, and you can count the total number of bboxes (group by their classes). 

Additionally, you would want to write a function to get rid the double counting too. Assuming you will see duplicated bboxes over the tile borders. I would write something like, if neighboring two bboxes' IOU > 80%, you will count them as one bbox instead of two. 

Hope this is helpful.
